streams/src/main/java/org/apache/kafka/streams/errors/DeserializationExceptionHandler.java

```diff
@@ -37,11 +37,27 @@ public interface DeserializationExceptionHandler extends Configurable {
      * @param context processor context
      * @param record record that failed deserialization
      * @param exception the actual exception
+     * @deprecated Please {@link #handle(ErrorHandlerContext, ConsumerRecord, Exception)}
```

Comment:
```suggestion
     * @deprecated Since 3.9. Use {@link #handle(ErrorHandlerContext, ConsumerRecord, Exception)} instead.
```

Reply:
Not sure about already merged PRs, but if they also have deprecation notice, this the the format we usually use. Including the version helps user to see how far behind they are and when the can expect the old API to be remove, so we want to include it.

Can you double check other PRs so we can update stuff if necessary?

Reply:
done
I wil check other PRs to be sure